docs/recipes/custom-audit/readme.md

```diff
@@ -1,11 +1,34 @@
-# Basic custom audit recipe for Lighthouse
+# Basic Custom Audit Recipe
 
-A hypothetical site measures the time from navigation start to when the page has initialized and the main search box is ready to be used. It saves that value in a global variable, `window.myLoadMetrics.searchableTime`.
+> **Tip**: see [Lighthouse Architecture](../../../docs/architecture.md) more information
+on terminology and architecture.
 
-This Lighthouse [gatherer](searchable-gatherer.js)/[audit](searchable-audit.js) pair will take that value from the context of the page and test whether or not it stays below a test threshold.
+## What this example does
 
-The config file tells Lighthouse where to find the gatherer and audit files, when to run them, and how to incorporate their output into the Lighthouse report.
+This example shows how to write a custom Lighthouse audit for a hypothetical site
+which measures the time from navigation start to when the page has initialized.
+The page is considered fully initialized when the main search box ("hero element")
+is ready to be used. The page saves that time in a global variable called `window.myLoadMetrics.searchableTime`.
 
-## Run
-With site running:
-`lighthouse --config-path=custom-config.js https://test-site.url`
+## The Audit, Gatherer, and Config
+
+[searchable-gatherer.js](searchable-gatherer.js) - a [Gatherer](https://github.com/GoogleChrome/lighthouse/blob/master/docs/architecture.md#components--terminology) that collects `window.myLoadMetrics.searchableTime`
+from the context of the page.
+
+[searchable-audit.js](searchable-audit.js) - an [Audit](https://github.com/GoogleChrome/lighthouse/blob/master/docs/architecture.md#components--terminology) that tests whether or not `window.myLoadMetrics.searchableTime`
+stays below a 4000ms. In other words, Lighthouse will consider the audit "passing"
+in the report if the search box initializes within 4s.
+
+[custom-config.js](custom-config.js) - this file tells Lighthouse where to
+find the gatherer and audit files, when to run them, and how to incorporate their
+output into the Lighthouse report. In this example, we have extended [Lighthouse's
+default configuration](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/default.js). If you're extending Lighthouse's default, passes with the same name are merged together, all other arrays will be concatenated, and primitive values will override the defaults.
```

Comment:
> It was moved from a section in the main readme

we should probably still include it somewhere. As paltry as it is, it's our only documentation of how extending a config works